database/sql/src/main/scala/cromwell/database/slick/tables/WorkflowMetadataSummaryEntryComponent.scala

```diff
@@ -49,6 +49,13 @@ trait WorkflowMetadataSummaryEntryComponent {
       if workflowMetadataSummaryEntry.workflowExecutionUuid === workflowExecutionUuid
     } yield workflowMetadataSummaryEntry)
 
+  val workflowMetadataSummaryEntryExistsForWorkflowExecutionUuid = Compiled(
+    (workflowExecutionUuid: Rep[String]) => (for {
+      summaryEntry <- workflowMetadataSummaryEntries
+      if summaryEntry.workflowExecutionUuid === workflowExecutionUuid
+    } yield summaryEntry).exists
```

Comment:
Doesn't `exists` potentially generate quite slow SQL? A count might be faster.

(Or, we might not care about performance here, which is perfectly valid.)